components/portfolio/AddCoOwner.tsx

```diff
@@ -0,0 +1,144 @@
+import { useTranslation } from '@/lib/i18n/init-client';
+import React from 'react'
+import { Button } from '../ui/button';
+import { EditIcon, Plus, Trash2 } from 'lucide-react';
+import { FormProvider, useForm, UseFormReturn } from 'react-hook-form';
+import { Dialog, DialogContent } from '@radix-ui/react-dialog';
+import { FormControl, FormField, FormItem } from '../ui/form';
+import { Input } from '../ui/input';
+import { DialogClose, DialogFooter, DialogHeader, DialogTitle } from '../ui/dialog';
+import { Label } from '../ui/label';
+import { AlertDialogAction, AlertDialogCancel, AlertDialogContent, AlertDialogDescription, AlertDialogHeader, AlertDialogTitle } from '../ui/alert-dialog';
+import { AlertDialog } from '@radix-ui/react-alert-dialog';
+import { z } from 'zod';
+
+const CoOwnerSchema = z.object({
+  email: z.string().email(),
+  title: z.string().min(1),
+})
+type CoOwerInfo = z.infer<typeof CoOwnerSchema>
+interface AddCoOwnerProps {
+  coownersForm: UseFormReturn<{
+    coowners: CoOwerInfo[]
+  }>
+  lang: string
+}
+function AddCoOwner({ lang, coownersForm }: AddCoOwnerProps) {
+  const { t } = useTranslation(lang, ["Portfolio", "ArtworkInfoStep"]);
+  const [coowners] = coownersForm.watch(["coowners"])
+  const [isOpen, setOpen] = React.useState(false)
+  const coownerForm = useForm<CoOwerInfo>()
+  const deleteCowner = (email: string) => {
+    coownersForm.setValue("coowners", coownersForm.getValues().coowners.filter((coowner) => coowner.email !== email))
+  }
```

Comment:
chỗ này hình như thiếu cái default value?
```typescript
function AddCoOwner({ lang, coownersForm }: AddCoOwnerProps) {
  const { t } = useTranslation(lang, ["Portfolio", "ArtworkInfoStep"]);
  const [coowners] = coownersForm.watch(["coowners"])
  const [isOpen, setOpen] = React.useState(false)
  const coownerForm = useForm<CoOwerInfo>({
    // Add default values to prevent controlled/uncontrolled switch
    defaultValues: {
      email: '',
      title: ''
    }
  })
  ...
}
```